pkg/config/remote/service/service.go

```diff
@@ -61,6 +63,8 @@ const (
 	initialFetchErrorLog uint64 = 5
 )
 
+var exportedMapStatus = expvar.NewMap("remoteConfigStatus")
```

Comment:
Could this be part of the `Service` struct too ?  The data in the map would be overwritten if a second `Service` is instanciated.

Reply:
I'll move it under `Service` 👍 
Note that `Service` should only be instanciated once

Reply:
Actually if the second service is instanciated (which is not supposed to happen), `expvar.NewMap` would panic
It seems that in the rest of the agent code, `NewMap` is often used outside of structs too https://github.com/DataDog/datadog-agent/blob/main/comp/forwarder/defaultforwarder/transaction/transaction.go#L29